pkg/cluster/adminupdate_test.go

```diff
@@ -130,7 +130,7 @@ func TestAdminUpdateSteps(t *testing.T) {
 				"[Action fixMCSCert-fm]",
 				"[Action fixMCSUserData-fm]",
 				"[Action ensureGatewayUpgrade-fm]",
-				"[Action rotateACRTokenPassword-fm]",
+				"[Condition rotateAndValidateACRTokenPassword-fm, timeout 5m0s]",
```

Comment:
Since we aren't waiting on a reconciliation here, do you think we should just make this an `Action` with no timeout instead of leaving it as `Condition`?